core/src/main/java/com/google/googlejavaformat/java/LineSeparator.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.googlejavaformat.java;
+
+import java.util.regex.Pattern;
+
+/** Line separator constants and utilities. */
+public enum LineSeparator {
+   WINDOWS("\r\n"),
+
+   UNIX("\n"),
+
+   MAC("\r");
```

Comment:
Another nit: even though at least one enum class e.g. `JavaFormatterOptions.Style` separates enums with additional lines, I wonder if in this case they'd read better without them. So something like

```java
WINDOWS("\r\n"),
UNIX("\n"),
MAC("\r");
```